CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## TBD
```

Comment:
Let's just call this version 2.3. We'll also need to update the version number in `setup.py`.